### PR TITLE
Fix RpmITCase + using ngnix package for CentOS 8

### LINKED ISSUE #7: RepoITCase doesn't test the functionality correctly
Try to run `RepoITCase` and read the logs. You will see that `dnf install` doesn't really install the RPM file. Something is broken. Has to be fixed.

### PR DESCRIPTION
**Pull request by Sammers21**
fixes #7

@yegor256, review the Pull Request please.

### CODE REVIEW

File: src/test/java/com/yegor256/rpm/RpmITCase.java [hunk 1]
```diff
@@ -105,10 +105,16 @@ public void savesAndLoads() throws Exception {
                     "cat /repo/repodata/primary.xml",
                     "cat /repo/repodata/filelists.xml",
                     "cat /repo/repodata/other.xml",
-                    "dnf config-manager --verbose --add-repo file:///repo",
+                    "echo '[repo]' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'name=test-repo' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'gpgcheck=0' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'enabled=1' >> /etc/yum.repos.d/repo.repo",
+                    "echo 'baseurl=file:///repo' >> /etc/yum.repos.d/repo.repo",
```
Comment (yegor256) [inline src/test/java/com/yegor256/rpm/RpmITCase.java]: @Sammers21 this is dirty. use `--setopt=<option>=<value>` instead: https://dnf-plugins-core.readthedocs.io/en/latest/config_manager.html

**Comment by yegor256** [comment]
@Sammers21 see above

File: src/test/java/com/yegor256/rpm/RpmITCase.java (hunk 1 above)
Comment (Sammers21) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]: I've tried and it does not work for me. Let's keep it the way it is now.

Yes, it's dirty, but it's more maintainable and easy to read

Comment (yegor256) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]: @Sammers21 what exactly `dnf config-manager` produces? Don't trust Artifactory too much, their team is far from being perfect

Comment (Sammers21) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]:
```
# dnf config-manager --add-repo=file:///repo123 --setopt=gpgcheck=0
# cat /etc/yum.repos.d/repo123.repo
[repo123]
name=created by dnf config-manager from file:///repo123
baseurl=file:///repo123
enabled=1
```
@yegor256 , as you can see, gpgcheck=0 property is not written to the file

Comment (yegor256) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]: @Sammers21 maybe this will help: https://bugzilla.redhat.com/show_bug.cgi?id=1253237

Comment (Sammers21) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]: Asked a question about it on SO: https://stackoverflow.com/questions/59486727/why-dnf-config-manager-save-does-not-work

Comment (Sammers21) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]: @yegor256 , would it be fine if we would generate `.repo` file with `dnf config-manager` and append `gpgcheck=0` to this file manually?

Comment (yegor256) [reply src/test/java/com/yegor256/rpm/RpmITCase.java]: @Sammers21 OK, let's do it this way

**Comment by yegor256** [comment]
@rultor merge

**Comment by Sammers21** [comment]
@yegor256 , i guess something wrong with @rultor

**Comment by yegor256** [comment]
@Sammers21 I think you should modify `.gitattributes` and mark `.rpm` as `binary` file

**Comment by Sammers21** [comment]
@yegor256, done

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**